metabase/metabase
# Add types generation for embedding SDK package

Relates to https://github.com/metabase/metabase/issues/40241

### Description

This adds TS types generation for Embedding SDK package

### How to verify

Check `Embedding SDK / sdk-build` job results.

Install latest @metabase/embedding-sdk-react pckage version (0.0.2-20240403-a0c32f6) to example app, run types check there.

### Checklist
